Automattic/wp-calypso
# Remove Unused Syncing Site Illustration

#### Changes proposed in this Pull Request

The illustration migration attempt has been a bit quiet for a while, but there's still plenty of illustrations that can be migrated. 🙂 

As mentioned [here](https://github.com/Automattic/wp-calypso/issues/36850#issuecomment-596816734), unused illustrations should be removed as part of #36850, though they could always be recovered in the future. This is one of them.

#### Testing instructions

Verify there's no uses of this illustration: https://github.com/Automattic/wp-calypso/search?q=%22al-syncing-site%22&unscoped_q=%22al-syncing-site%22

Part of #36850
